flight/utils/movement_controller.py

```diff
@@ -95,25 +96,52 @@ async def move_to(self, drone: System, pylon: LatLon) -> bool:
                 return True
             count += 1
 
-    async def turn(self, drone: System) -> bool:
+    async def turn(self, drone: System, degrees_turned = 180: int, left_turn = True: bool, time = 3.5: float) -> bool:
```

Comment:
The type annotations need to be changed here such that the type comes before the default parameter:
```
async def turn(self, drone: System, degrees_turned: int = 180, left_turn: bool = True, time: float = 3.5) -> bool:
```

Reply:
Additionally, the new parameters should be described in the docstring (along with the existing `drone` description)